charts/camunda-platform/values.yaml

```diff
@@ -1746,6 +1746,105 @@ web-modeler:
       # Postgresql.auth.database defines the name of the database to be created for Web Modeler
       database: web-modeler
 
+# Connectors configuration for the Connectors.
+connectors:
```

Comment:
The readiness, liveness, and startup probs are not there (we can do it later only if they are not implemented in the Connectors yet).

Reply:
They are not implemented yet.

Reply:
Put a task at backlog
https://github.com/camunda/connectors-bundle/issues/348